omniport/core/session_auth/http_urls.py

```diff
@@ -2,10 +2,13 @@
 
 from session_auth.views.login import Login
 from session_auth.views.logout import Logout
+from session_auth.views.illustration_roulette import IllustrationRoulette
 
 app_name = 'session_auth'
 
 urlpatterns = [
     path('login/', Login.as_view(), name='login'),
     path('logout/', Logout.as_view(), name='logout'),
+    path('illustration_roulette/', IllustrationRoulette.as_view(),
+         name='illustration_roulette'),
```

Comment:
We usually split newlines differently.
```python
    path(
        'illustration_roulette/',
        IllustrationRoulette.as_view(),
        name='illustration_roulette'
    )
```
And if one view doesn't fit in one line, break all the above in the same way.

Reply:
@pradumangoyal This is quite necessary.